Sources/FuzzilliCli/Profiles/XSProfile.swift

```diff
@@ -44,21 +338,38 @@ let xsProfile = Profile(
         ("fuzzilli('FUZZILLI_CRASH', 2)", .shouldCrash),
     ],
 
-    additionalCodeGenerators: [],
+    additionalCodeGenerators: [
+        (StressXSMemoryFail,    5),
+        (StressXSGC,    5),
+        (HardenGenerator, 5),
+        (CompartmentGenerator, 5),
+        (UnicodeStringGenerator, 2),
+        (ModuleSourceGenerator, 3)
+    ],
 
-    additionalProgramTemplates: WeightedList<ProgramTemplate>([]),
+    additionalProgramTemplates: WeightedList<ProgramTemplate>([
+        (RegExpFuzzer, 1),
+    ]),
 
     disabledCodeGenerators: [],
 
     disabledMutators: [],
 
     additionalBuiltins: [
         "gc"                  : .function([] => .undefined),
+        "memoryFail"          : .function([.number] => .number),
         "print"               : .function([.string] => .undefined),
-        "placeholder"         : .function([] => .undefined),
+
+        // hardened javascript
+        "Compartment"         : .function([] => .jsCompartmentConstructor),
```

Comment:
Same here as with the signatures in the object groups, please provide the types that these constructors expect.